assets/airports/rjbb.json

```diff
@@ -0,0 +1,2732 @@
+                "06R": ["B6R10", ["B6R11", "A25+"], "HELEN", "MAIKO"],
+                "24L": ["B4L10", ["BB430", "S210"],["HELEN", "A80+"], "MAIKO"],
+                "24R": [["BB430", "S210"], ["HELEN", "A80+"], "MAIKO"]
+            },
+            "draw": [
+                ["HELEN", "MAIKO"],
+                ["B6R11", "HELEN"],
+                ["BB430", "HELEN"],
+                ["MAIKO", "WENDY", "SHION*"],
+                ["SHION", "SOUJA*"],
+                ["SHION", "WASYU*"],
+                ["SHION", "HABAR*"],
+                ["CHIZU*"]
+            ]
+        },
+        "IWAYA1": {
+            "icao": "IWAYA1",
+            "name": "Iwaya One",
+            "exitPoints": {
+                "SHION": [
+					"WENDY", "SHION"
+				],
+                "SOUJA": [
+					"WENDY", "SHION", "SOUJA"
+				],
+                "WASYU": [
+					"WENDY", "SHION", "WASYU"
+				],
+                "HABAR": [
+					"WENDY", "SHION", "HABAR"
+				],
+                "CHIZU": [
+					"KAWAT", "CHIZU"
+				]
+            },
+            "rwy": {
+                "24L": ["BB430", ["HELEN", "A80+"], "MAIKO"],
+                "24R": ["BB430", ["HELEN", "A80+"], "MAIKO"]
+            },
+            "draw": [
+                ["HELEN"],
+                ["SHION*"],
+                ["SOUJA*"],
+                ["WASYU*"],
+                ["HABAR*"],
+                ["CHIZU*"]
+            ]
+        },
+        "DAISY1": {
+            "icao": "DAISY1",
+            "name": "Daisy One",
+            "exitPoints": {
+                "HIBIS": [
+					"HIBIS"
+				]
+            },
+            "rwy": {
+                "06L": ["DAISY", ["JULIA", "A80+"], "LINDA", "GLORY"],
+                "06R": ["B6R10", ["B6R11", "A25+"], "B6R12", "DAISY", ["JULIA", "A80+"], "LINDA", "GLORY"],
+                "24L": ["B4L10", ["BB430", "S210"], ["DAISY", "A60"], ["JULIA", "A80+"], "LINDA", "GLORY"],
+                "24R": [["DAISY", "A60+"], ["JULIA", "A80+"], "LINDA", "GLORY"]
+            },
+            "draw": [
+                ["DAISY", "JULIA", "LINDA", "GLORY"],
+                ["B6R11", "B6R12", "DAISY"],
+                ["BB430", "DAISY"],
+                ["GLORY", "HIBIS*"]
+            ]
+        }
+    },
+    "stars": {
+        "AKASI": {
+            "icao": "AKASI",
+            "name": "Akasi",
+            "entryPoints": {
+                "TOKUSHIMA": [
+					"TSC"
+				]
+            },
+            "body": ["AKASI", "MAIKO", ["MAYAH", "A40"]],
+            "draw": [
+                ["TSC*", "AKASI", "MAIKO", "MAYAH"]
+            ]
+        },
+        "GATES": {
+            "icao": "GATES",
+            "name": "Gates",
+            "entryPoints": {
+                "TOKUSHIMA": [
+					"TSC"
+				]
+            },
+            "body": [
+                ["GATES", "A40"]
+            ],
+            "draw": [
+                ["TSC*", "GATES"]
+            ]
+        },
+        "DANDE_A": {
```

Comment:
What's the deal with these variants? Provided that suffixes be completed, is that something that would cover this?

Regardless, due to the need for users to type the names of these procedures, please refrain from using special characters like underscores.